src/darwin/Framework/CHIP/MTRDeviceControllerFactory.mm

```diff
@@ -845,4 +846,24 @@ - (void)setStartServer:(BOOL)startServer
     self.shouldStartServer = startServer;
 }
 
+- (nullable NSArray<NSData *> *)paaCerts
```

Comment:
Are these backwards compatibility only? If so should they be marked as deprecated in some way?

Reply:
They're for compat only, and they're marked deprecated in the header.